site/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -50,8 +56,33 @@ can gather data on requests that failed as a result of
 `Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy`.
 
 If you don't think you can make these changes in time for Chrome 91, you can
-[register for an origin trial](#origin-trial) to retain current Desktop
-Chrome behavior until Chrome 93.
+[register for an origin trial](#origin-trial) to retain current Desktop Chrome
+behavior until at least Chrome 96.
+
+{% Aside %}
+**Update, April 2021**
+
+We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
+cross-origin isolation requires all subresources to explicitly opt-in. And we
+have come up with the idea of going in the opposite direction: a new [COEP
+"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
+allows loading resources without the CORP header by stripping all their
+credentials. We are figuring out the details of how it should work, but we hope
+this will lighten your burden of making sure the subresources are sending the
+`Cross-Origin-Resource-Policy` header.
+
+Also, it's known that the `Cross-Origin-Opener-Policy: same-origin` header will
+break integrations that require cross-origin window interactions such as OAuth
+and payments. To mitigate this problem, we are [exploring relaxing the
+condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
+isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
+the communication with the window opened by itself will be possible.
+
+If you want to enable cross-origin isolation to use `SharedArrayBuffer` but are
+blocked by these challenges, we recommend [registering for an origin
+trial](#origin-trial) and waiting until the new modes are available. We are not
+planning to terminate the origin trial until these new modes are available. {%
```

Comment:
can you put endAside on its own line?

Reply:
That should probably be handled in a separate PR as this content has already been merged into main.